tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -0,0 +1,357 @@
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_retrieve_rhev_vm_list(self):
+        """Retrieve the Virtual machine list from rhev compute resource
+
+        @id: f8cef7fb-e14c-4d12-9862-0e448a59ca50
+
+        @setup: rhev hostname and credentials.
+
+        @steps:
+        1. Select the created compute resource.
+        2. Go to "Virtual Machines" tab.
+
+        @Assert: The Virtual machines should be displayed
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_rhev_with_image(self):
+        """ Provision a host on rhev compute resource with image based
+
+        @id: 80abd6b1-31cd-4f3e-949c-f1ca608d0bbb
+
+        @setup: rhev hostname and credentials.
+        1. Configured subnet for provisioning of the host.
+        2. Configured domains for the host.
+        3. Population of images into satellite from rhev templates.
+        4. Activation key and CV for the host.
+
+        @steps:
+        1. Go to "Hosts --> New host".
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select rhev compute resource from "Deploy on" drop down.
+        4. Associate appropriate feature capsules.
+        5. Go to "operating system tab".
+        6. Edit Provisioning Method to image based.
+        7. Select the appropriate image .
+        8. Associate the activation key and submit.
+
+        @Assert: The host should be provisioned successfully
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_rhev_with_compute_profile(self):
+        """ Provision a host on rhev compute resource with compute profile
+        default (3-Large)
+
+        @id: fe4a05ef-d548-4c28-80d0-d17851fb4b03
+
+        @setup: rhev hostname ,credentials and provisioning setup.
+
+        @steps:
+        1. Go to "Hosts --> New host"
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select rhev compute resource from "Deploy on" drop down.
+        4. Select the "Compute profile" from the drop down.
+        5. Provision the host using the compute profile.
+
+        @Assert: The host should be provisioned successfully
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_rhev_with_custom_compute_settings(self):
+        """ Provision a host on rhev compute resource with
+         custom disk, cpu count and memory.
+
+        @id: a972c095-7567-4bb0-86cb-9bd835fed7b7
+
+        @setup: rhev hostname ,credentials and provisioning setup.
+
+        @steps:
+        1. Go to "Hosts --> New host"
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select rhev custom compute resource from "Deploy on" drop down.
+        4. Select the "Compute profile" from the drop down.
```

Comment:
Select the custom "compute profile" with given disk size, cpu count and memory ?